src/components/_classes/nested/NestedComponent.js

```diff
@@ -728,7 +728,7 @@ export default class NestedComponent extends Field {
     if (component.type === 'button') {
       return false;
     }
-    if (value && component.hasValue(value)) {
+    if (value && component.hasValue(value) && component.component.type !== 'tabs') {
```

Comment:
This code should be within the Tabs component which derives from NestedComponent. You need to create a method in there that can be overridden. Maybe have "hasValue" return false?

Reply:
This issue already fixed by this PR: https://github.com/formio/formio.js/pull/4812.
So we don't need my changes, this PR can be closed.